package.json

```diff
@@ -81,6 +84,7 @@
     "@babel/parser": "^7.16.6",
     "@babel/traverse": "^7.16.5",
     "@microsoft/jest-sarif": "^1.0.0-beta.0",
+    "@types/jest": "^27.0.3",
```

Comment:
this was added because I run tsserver on js files, and have automatic type acquisition disabled -- I think it's it's useful to have